src/generate-types.ts

```diff
@@ -625,15 +625,11 @@ export function run(options: Options) {
     );
     const brand = ts.createExpressionWithTypeArguments(
       [
-        ts.createTypeReferenceNode(ts.createIdentifier('ShapeOf' + oautil.typenamify(key)), []),
+        ts.createTypeReferenceNode(ts.createIdentifier('never'), []),
```

Comment:
Do we still need to give the Shape then? Or are we keeping it for backwards compatibility reasons?

Reply:
backwards compatibility. We already cause trouble with the branding change so lets spread the pain a bit.